classes/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = nil
```

Comment:
- [ ] As required for the project, I think it would be better if you update the value of your `:id` by using the `Random` method. This method will help you to create random `id` when you create a new person. You can update to this:
```ruby
  ...
  def initialize(age, name = 'Unknown', parent_permission: true)
    @id = Random.rand(1..1000)
   ...
  end
  ...
```